package main

func main() {
	selfError()
}
